code/game/machinery/vending/vendor_types/crew/commanding_officer.dm

```diff
@@ -116,3 +120,8 @@ GLOBAL_LIST_INIT(cm_vending_clothing_commanding_officer, list(
 	spawned_gear_list = list(
 		/obj/item/weapon/gun/rifle/m46c,
 	)
+
+/obj/effect/essentials_set/co/shotgunpreset
+	spawned_gear_list = list(
+		/obj/item/weapon/gun/rifle/xm52,
+	)
```

Comment:
```suggestion
/obj/effect/essentials_set/co/shotgunpreset
	spawned_gear_list = list(
		/obj/item/weapon/gun/rifle/xm52,
		/obj/item/storage/box/guncase/xm52,
	)
/obj/item/storage/box/guncase/xm52
	name = "\improper XM52 ammunition case"
	desc = "A case containing ammunition for an XM52."
	can_hold = list(/obj/item/ammo_magazine/rifle/xm52, /obj/item/ammo_magazine/handful/shotgun/heavy/buckshot)
	storage_slots = 6

/obj/item/storage/box/guncase/xm52/fill_preset_inventory()
	new /obj/item/ammo_magazine/rifle/xm52(src)
	new /obj/item/ammo_magazine/rifle/xm52(src)
	new /obj/item/ammo_magazine/rifle/xm52/flechette(src)
	new /obj/item/ammo_magazine/rifle/xm52/flechette(src)
	new /obj/item/ammo_magazine/handful/shotgun/heavy/buckshot(src)
	new /obj/item/ammo_magazine/handful/shotgun/heavy/buckshot(src)
```
As there is no way outside of this vendor to get ammo for the XM52, it should start with some.